arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -906,24 +999,80 @@
           }
         ],
         "responses": {
-          "202": {
-            "description": "Accepted the request to deactivate the device."
+          "200": {
+            "description": "The Alert settings.",
+            "schema": {
+              "$ref": "#/definitions/AlertSettings"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "DeviceSettings"
+        ],
+        "description": "Creates or updates the alert settings of the specified device.",
+        "operationId": "DeviceSettings_CreateOrUpdateAlertSettings",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "deviceName",
```

Comment:
you may want to move this parameter after the "managerNameParameter" so method signature in code generation follows the same order as the parameters in the path.